onnxruntime/test/testdata/custom_op_get_const_input_test_library/custom_op.cc

```diff
@@ -5,7 +5,12 @@
 
 void simple_assert(const bool cond, const std::string& text) {
   if (!cond) {
+#ifndef ORT_NO_EXCEPTIONS
```

Comment:
there are some macros that we can use for exception-related code across build types:
https://github.com/microsoft/onnxruntime/blob/322237f482c94578cabbbdd1bf853417766aa875/include/onnxruntime/core/common/common.h#L98-L183

Reply:
The way this custom op library is implemented, it seems like it is a standalone library (that is linked dynamically):

https://github.com/microsoft/onnxruntime/blob/322237f482c94578cabbbdd1bf853417766aa875/cmake/onnxruntime_unittests.cmake#L1569-L1598

It doesn't take any linker dependencies on ORT. I can add the definitions in case we want to do that.

Reply:
So, ORT_TRY and ORT_CATCH would work. But ORT_THROW probably needs those undefined symbols to be defined (like `PrintFinalMessage` and `GetStacktrace`).

Let me add those and try.

Reply:
ah, maybe not worth the trouble then. I saw other usage of common.h (ORT_UNUSED_PARAMETER) and thought it would be ok.